webpack/scenes/ContentViews/Details/Filters/Rules/Package/AddEditPackageRuleModal.js

```diff
@@ -62,7 +62,7 @@ const AddEditPackageRuleModal = ({ filterId, onClose, selectedFilterRuleData })
 
   const formVersionParams = () => {
     switch (versionComparator) {
-      case 'All Versions':
+      case 'All versions':
```

Comment:
Don't know when this changed but the dropdown selector for these options doesn't display properly in the form. Probably cause modal is set to high zIndex to not hide between nav..
We should convert the Select and SelectOption to FormSelect and FormSelectOption..
```
          <FormSelect
            value={versionComparator}
            onChange={onSelect}
            id="version_comparator"
            name="version_comparator"
            aria-label="version_comparator"
          >
            {
              Object.values(VersionModifiers).map(item => (
                <FormSelectOption key={item} value={item} label={VersionModifiers[item]}>
                </FormSelectOption>))
            }
          </FormSelect>
```

What do you think?

Reply:
I can do that!